app/controllers/licences.controller.js

```diff
@@ -25,9 +25,9 @@ async function returnsRequired (request, h) {
 }
 
 async function view (request, h) {
-  const { id } = request.params
+  const { params: { id }, auth } = request
 
-  const data = await ViewLicenceService.go(id)
+  const data = await ViewLicenceService.go(id, auth)
```

Comment:
I could not see the benefit adding to this test as the ViewLicenceService is mocked. 

I can not see other places where the the mocks args are tested in the rest of the code base. 

I think the presenter test is sufficient to test the auth functionality but happy to discuss.

Reply:
Agreed. This is why our controllers are 'dumb' and leave all the work to the services. In general, we limit our controller tests to 'when the request is valid it [returns x/redirects to y]' and 'when request is invalid it [returns z]'

Any other tests of behaviour are done directly against the service/presenter.